src/components/Button/Button.tsx

```diff
@@ -0,0 +1,64 @@
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import React from "react";
+
+import { BaseButtonProps } from "./BaseButton";
+
+import "lbh-frontend/lbh/components/lbh-button/_button.scss";
+
+/**
+ * The proptypes for {@link Button }
+ */
+export interface ButtonProps extends BaseButtonProps {
+  buttonName?: string;
+  buttonType?: React.ButtonHTMLAttributes<HTMLButtonElement>["type"];
```

Comment:
Feels like these could be `name` and `type` instead of `buttonName` and `buttonType` for terseness.